avalanched-aws/src/agent/mod.rs

```diff
@@ -1702,6 +1732,7 @@ async fn monitor_spot_instance_action(
 
         let shared_config = aws_manager::load_config(
             Some(local_region.to_string()),
+            Some(profile_name.clone()),
```

Comment:
Let's use None, because the profile name in spec, might be different than the one we use in remote machines.